patches.json

```diff
@@ -101,5 +101,132 @@
         }
       }
     }
+  },
+  "MDVA-30195": {
+    "magento/magento2-base": {
+      "Fixes the issue where cron jobs fail if database name is too long, resulting in categories not being updated on the frontend.": {
+        ">=2.3.1 <=2.3.4-p2": {
+          "file": "os/MDVA-30195__fix_to_handle_long_database_names__2.3.4-p2.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30106": {
+    "magento/magento2-base": {
+      "Fixes an issue where during checkout payments are not loaded with “Cannot read property ‘length’ of null ” error in JS console.": {
+        ">=2.3.0 <=2.4.0": {
+          "file": "os/MDVA-30106__fix_validation_for_possible_null_objects_on_validation_rules__2.3.4.patch"
+        }
+      }
+    }
+  },
+  "MDVA-28656": {
+    "magento/magento2-base": {
+      "Fixes the issue where if an order was placed with no payment information required (for example, with 100% discount) and an invoice was created for the order, the order status changes to Closed instead of Complete.": {
+        ">=2.3.1 <=2.4.0": {
```

Comment:
This fix is included in Magento 2.3.6  (https://jira.corp.magento.com/browse/MC-35554), so version constraint should be ">=2.3.1 <2.3.6 || 2.4.0"

Reply:
fixed